### PR TITLE
Update generating-fsharp-types-from-edmx.md

### PR DESCRIPTION
**Pull request by GoFightNguyen**
# Adding link to the Configuring the Entity Data Model section

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1136](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx?branch=pr-en-us-1136)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C12%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610121326077696-1136%5Cworkflow_report.html).

File: docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ If you already have an EDMX file, you can skip this step.
 
 #### To create an EDMX file
 
-- If you don't already have an EDMX file, you can follow the instructions at the end of this walkthrough in the step **To configure the Entity Data Model**.
+- If you don't already have an EDMX file, you can follow the instructions at the end of this walkthrough in the step [Configuring the Entity Data Model](#configuring-the-entity-data-model).
```
Comment (cartermp) [inline docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md]: > `#configuring-the-entity-data-model`

This should be `generating-fsharp-types-from-edmx.md#configuring-the-entity-data-model.md`.

**Comment by mairaw** [comment]
Should I close this PR @jasonsnguyen? I see you've created a new one for the same topic in #1137. 
Do you know that you can make changes to the same branch you were working on and the commits will automatically show up on the PR?

**Comment by jasonsnguyen** [comment]
@mairaw Yes, you can close this pull request. My pull request in topic 1137 does not actually include this, but I can add it another time.

I just learned today that I can make changes to the same branch I was working on and then those commits show up automatically. Thanks

**Pull request closed without merging**